lib/adapter.js

```diff
@@ -37,74 +41,83 @@ class CucumberAdapter {
         this.capabilities = capabilities
 
         this.cucumberOpts = Object.assign(DEFAULT_OPTS, config.cucumberOpts)
```

Comment:
I guess this is a question for @christian-bromann 
Do we want to allow globs in the require part of the wdio.conf? 

Also, we could use similar code (with glob) to automatically import files from step_definitions/ folder, the way cucumber (and current wdio-cucumber-framework implementation) works, as far as I understand